docker_startup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -it --rm -p 4000:4000 -v $PWD:/src jekyll/jekyll jekyll server -w -s /src
```

Comment:
Two quick comments to align with the command in `.travis.yml`:

- the current directory could be mounted as `-v $PWD:/srv/jekyll` which is the default Jekyll location of the Docker image and should prevent the need for the `-s /src` option
- the `pages` tag of the Jekyll image could be used instead of the default tag